dataline-scheduler/src/main/java/io/dataline/scheduler/SchedulerApp.java

```diff
@@ -54,8 +54,7 @@
 
   private static final int MAX_WORKERS = 4;
   private static final long JOB_SUBMITTER_DELAY_MILLIS = 1000L;
-  private static final ThreadFactory THREAD_FACTORY =
-      new ThreadFactoryBuilder().setNameFormat("scheduler-%d").build();
+  private static final ThreadFactory THREAD_FACTORY = new ThreadFactoryBuilder().setNameFormat("scheduler-%d").build();
```

Comment:
Not really related to this PR but shoudln't the name be what it is as opposed to what created it? If so then this should be called "worker"